### ISSUE TITLE
Re-enable the creation of statelite when copycds on RHEL 7

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#2025]
Currently when we copycds on RHEL operating systems, we create the 
- install-compute
- netboot-compute
- install-service

Would like to be able to have a osimage created for statelite to start testing that. 

Suggestion is perhaps adding an option to copycds with "statelite" and it will create the osimage from the template files that xCAT ships...

**Comment by gurevichmark** [comment]
Perhaps it can use a `--plus` flag similar to `chdef --plus`. This way user can say something like `copycds --plus statelite` to create a statelite image in addition to the 3 standard ones: "install-compute, netboot-compute, install-service"

**Comment by whowutwut** [comment]
@bybai  Similar to what  @gurevichmark suggested but not using the copycds command... would it be of value if we provided some utility that would create/remove osimage templates based on template files that we already ship... like kvm, statelite, etc...    this would be executed after ISO has already been copied into xCAT. 

If the osimage already exists, we would not create it.  But if it doesn't exist, we would create it in with default values.

**Comment by tingtli** [comment]
@whowutwut  Just for your test, not from template, but from xcat commands.
You can create statelite osimage definition using following command. Take rhels7.2 statelite as an example.
mkdef -t osimage -o rhels7.2-ppc64-statelite-compute -u profile=compute provmethod=statelite osvers=rhels7.2

**Comment by immarvin** [comment]
hi @whowutwut , 
the creation of satellite osimage was removed in the commit:
````
commit db8a4dd07dc5fa40921b96b9c9f87384b6193345
Author: immarvin <yangsbj@cn.ibm.com>
Date:   Tue Jun 2 02:02:40 2015 -0400

    donot create statelite osimages and diskless service osimages after copycds,

````

if satellite become important for the user, I think we can simply get it back , at lease we can get "<osver>-<arch>-statelite-compute"  osimage back on copycds.  since the creation of osimage involves the search/match of template, pkglist and otherpkglist files in a waterfall model, I think it should better to recover the logic in "copycds"

**Comment by whowutwut** [comment]
@tingtli  thanks for that step.  Will give it a try

**Comment by bybai** [comment]
Hi @whowutwut ,

I agreed with Yang song 's opinion, we can just get statelite back.  It is no need to add additional option for copycds. What is your opinion?

**Comment by whowutwut** [comment]
@bybai @immarvin Ok, this is OK for now, but what is xCAT's stance for support on this?  The statelite image is hidden and not created by default during copycds....   does that mean we are deprecating the support for statelite?  If so, then we need to bring it back.

**Comment by bybai** [comment]
Hi @whowutwut ,
I will bring it back, thanks.

**Comment by immarvin** [comment]
fixed in https://github.com/xcat2/xcat-core/pull/2178

**Comment by bybai** [comment]
fixed in #2178
